week-1-practise/level-1/8-function-output-1.js

```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
+// - What does this program do? -it takes random number between  1.0....9.0
```

Comment:
This program generates random numbers and it doesn't take any parameter. try to answer all the questions as this questions are really helpful for you to understand the concept.